src/fenwicktree.rs

```diff
@@ -1,35 +1,37 @@
+use std::iter::repeat_with;
+
+use crate::num_traits::Zero;
```

Comment:
- https://github.com/rust-lang-ja/ac-library-rs/pull/103 : Implement `From<Vec<T>>` and `FromIterator` for `FenwickTree`
- https://github.com/rust-lang-ja/ac-library-rs/pull/104 : Avoid `Clone` in `FenwickTree`

These appear to be changes that need not specifically rely on #102. Check out the application examples below.

https://github.com/rust-lang-ja/ac-library-rs/compare/7e99fd2941976764b4e7925898fae7cc77890885...mizar:ac-library-rs:4be604494341a86242b77120e5685740dc41791f?expand=1

Reply:
I don't think requiring using `Default` is not always a good idea because the semantics are different.  `Zero` traits is clearly for the "additive identity", while `Default` trait is for a general "default" instance.  The latter is not guaranteed to be an additive identity, and that restriction shall be stated in the doc instead of as a type restriction.  I prefer type restrictions to other document constraints, and thus thought introducing `Zero` trait would be reasonable.